ios/FluentUI.Demo/FluentUI.Demo/Demos/PeoplePickerDemoController.swift

```diff
@@ -47,7 +47,7 @@ class PeoplePickerDemoController: DemoController {
             addDescription(text: variant.description)
             addPeoplePicker(for: variant)
             if index != PeoplePickerSampleData.variants.count - 1 {
-                container.addArrangedSubview(Separator())
+                container.addArrangedSubview(MSFDivider().view)
```

Comment:
For each of these instances in UIKit, the created `MSFDivider` needs to be stored somewhere, as it's responsible for the updating and lifecycle of its view.

Reply:
This extends to `BadgeFieldDemoController`, `ColorDemoController`, `ShimmerLinesViewDemoController`, `TableViewHeaderFooterViewDemoController`, and `BottomCommandingController`.